exchange/bidder.go

```diff
@@ -522,11 +523,27 @@ func (bidder *bidderAdapter) doRequest(ctx context.Context, req *adapters.Reques
 
 func (bidder *bidderAdapter) doRequestImpl(ctx context.Context, req *adapters.RequestData, logger util.LogMsg, bidderRequestStartTime time.Time, tmaxAdjustments *TmaxAdjustmentsPreprocessed) *httpCallInfo {
 	var requestBody []byte
+	var gzipWriterPool = sync.Pool{
+		New: func() interface{} {
+			return gzip.NewWriter(nil)
+		},
+	}
```

Comment:
This isn't going to work the same as it does in the benchmark test because the pool is created per `doRequestImpl` call. You need to move the pool creation up a level so that it's only created once.